src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -111,6 +111,13 @@ <h3 id="history">
       {% include "reviewers/includes/paginator_history.html" %}
     </div>
   </div>
+  <div id="unreviewed-other-queue">
+    {% if has_versions_with_due_date_in_other_queue %}
+      <p class="unreviewed-versions-warning" style="color: red;">
+        This add-on has 1 or more versions with a due date in other pages.
```

Comment:
Can you include a screenshot of what the page looks like?

Reply:
![image](https://github.com/mozilla/addons-server/assets/63402349/78f196f1-a754-4ad1-891c-2dbd19c418bf)